### PR TITLE
[CHK-562] feat(opex): transaction-service opex dashboard configurations file

### PR DESCRIPTION
**Pull request by ciuffagianluca**
Add opex dashboard configurations file

#### List of Changes

Add .opex and .identity folder
Add create-dashboard yaml file

#### Motivation and Context

Monitoring purpose

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .identity/02_application_main_auth.tf [hunk 1]
```diff
@@ -0,0 +1,17 @@
+resource "azurerm_role_assignment" "main_terraform_subscription" {
+  scope                = data.azurerm_subscription.current.id
+  role_definition_name = "Reader"
+  principal_id         = azuread_service_principal.main.object_id
+}
+
+resource "azurerm_role_assignment" "main_terraform_storage_account_tfstate_app" {
```
Comment (infantesimone) [inline .identity/02_application_main_auth.tf:7]: do we want to remove it and add it only if it's needed?

**Review by infantesimone — APPROVED** [review]
just a comment

File: .identity/02_application_main_auth.tf (hunk 1 above)
Comment (ciuffagianluca) [reply .identity/02_application_main_auth.tf:7]: ok, i do it

**Comment by ciuffagianluca** [comment]
> "azurerm_role_assignment" "main_terraform_storage_account_tfstate_app"

Removed unuseful role assignment

**Review by giovanniberti — APPROVED** [review]

**Review by infantesimone — APPROVED** [review]

**Review by infantesimone — APPROVED** [review]

**Pull request merged**